src/docs/deployment/android.md

```diff
@@ -245,3 +291,4 @@ Google Play Store, see the [Google Play publishing documentation][play].
 [launchericons]: {{site.android-dev}}/guide/practices/ui_guidelines/icon_design_launcher
 [configuration qualifiers]: {{site.android-dev}}/guide/topics/resources/providing-resources#AlternativeResources
 [play]: {{site.android-dev}}/distribute/googleplay/start
+[Issue 18494]: https://github.com/flutter/flutter/issues/18494
```

Comment:
Out of curiosity, why did this link end up down here, but most of the other new ones did not?